src/main/java/hng_java_boilerplate/externalPages/controller/AboutPageController.java

```diff
@@ -0,0 +1,66 @@
+package hng_java_boilerplate.externalPages.controller;
+
+import hng_java_boilerplate.externalPages.dtos.AboutPageDTO;
+import hng_java_boilerplate.externalPages.dtos.RetrievalDTO;
+import hng_java_boilerplate.externalPages.services.AboutPageService;
+import hng_java_boilerplate.user.enums.Role;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.security.core.Authentication;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Collections;
+
+@RequiredArgsConstructor
+@RestController("/api/v1/content/about")
```

Comment:
Did you run your code? The base url should be on RequestMapping and not RestController

Reply:
I have effected this change